week1/gallery.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>숭이의 동물원</title>
+  </head>
+  <body>
+    <header id="title">🤍숭이의 동물원🤍</header>
```

Comment:
header 는 제목, 로고, 검색 폼, 작성자 이름 등의 요소 를 포함하는 소개 및 탐색에 도움을 주는 콘텐츠를 담는 영역!이기 때문에 정확히는 이 안에서 
```suggestion
      <header id="title"><h1>🤍숭이의 동물원🤍</h1></header>
```
이렇게 써주는게 좋답니다! :)